src/com/serotonin/mango/rt/dataImage/PointValueCache.java

```diff
@@ -35,104 +34,135 @@
 public class PointValueCache {
```

Comment:
The cache should contain the only structure or data source for what is cached, there should be no DAO dependency. It should be in DAO that the use of Cache should be sewn, preferably in the form of a separate class, I think that in the future we will try to do it properly, probably using an external library.